src/components/OSCALSsp.js

```diff
@@ -73,7 +78,14 @@ export default function OSCALSsp(props) {
 
   return (
     <div className={classes.paper}>
-      <OSCALMetadata metadata={ssp.metadata} />
+      <OSCALMetadata
+        metadata={ssp.metadata}
+        isEditable={props.isEditable}
+        editedField={["system-security-plan"]}
```

Comment:
I thought we were able to drop `editedField` since we have it in `patchData`?

Reply:
We can and that should be dropped.